actions.go

```diff
@@ -419,31 +419,34 @@ type GetenvFunc func(key string) string
 
 // Context of current workflow
 type Context struct {
-	Event      map[string]interface{}
-	EventName  string
-	SHA        string
-	Ref        string
-	Workflow   string
-	Action     string
-	Actor      string
-	Job        string
-	RunNumber  int
-	RunID      int
-	APIURL     string
-	ServerURL  string
-	GraphqlURL string
+	Event            map[string]interface{}
+	EventPayloadPath string `env:"GITHUB_EVENT_PATH,required"`
+	EventName        string
+	SHA              string `env:"GITHUB_SHA"`
+	Ref              string `env:"GITHUB_REF"`
+	Workflow         string `env:"GITHUB_WORKFLOW"`
+	Action           string `env:"GITHUB_ACTION"`
+	Actor            string `env:"GITHUB_ACTOR"`
+	Job              string `env:"GITHUB_JOB"`
+	RunNumber        int    `env:"GITHUB_RUN_NUMBER"`
+	RunID            int    `env:"GITHUB_RUN_ID"`
+	APIURL           string `env:"GITHUB_API_URL,default=https://api.github.com"`
```

Comment:
Is there a reason to have defaults? I feel like, if the environment doesn't inject these values, we shouldn't either. What does the Node SDK do here?

Reply:
We're replacating more or less the same logic: https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts#L49-L52

Reply:
sure